src/app/components/fileupload/fileupload.ts

```diff
@@ -901,4 +901,4 @@ export class FileUpload implements AfterViewInit, AfterContentInit, OnInit, OnDe
     exports: [FileUpload, SharedModule, ButtonModule, ProgressBarModule, MessagesModule],
     declarations: [FileUpload]
 })
-export class FileUploadModule { }
+export class FileUploadModule {}
```

Comment:
@denny99  if you didn't work this file then is it need to change or remove this file any line ?

Reply:
Otherwise the Pull Request Prettier Stage fails, so i just ran the prettier command to auto format the file

Reply:
@denny99  Good point can you mention it your PR description so PR reviewer can understand the purpose of those extra changes.

Reply:
Like in my very first comment above :D 
Yeah sure i can copy paste that into the description

Reply:
done